ui/shared/components/buttons/UpdateButton.jsx

```diff
@@ -8,35 +8,39 @@ import Modal from '../modal/Modal'
 const UpdateButton = (
   { onSubmit, initialValues, formFields, modalTitle, modalId, buttonText, editIconName, size, modalSize, showErrorPanel,
     disabled, confirmDialog, submitButtonText, buttonFloated, trigger, formContainer = <div /> },
-) =>
-  <Modal
-    title={modalTitle}
-    modalName={modalId}
-    size={modalSize}
-    trigger={trigger || (
-      <ButtonLink
-        content={buttonText}
-        icon={editIconName || 'write'}
-        labelPosition={buttonText && 'right'}
-        size={size}
-        disabled={disabled}
-        floated={buttonFloated}
-      />
-    )}
-  >
-    {React.cloneElement(formContainer, { children: (
-      <ReduxFormWrapper
-        onSubmit={onSubmit}
-        form={modalId}
-        initialValues={initialValues}
-        fields={formFields}
-        showErrorPanel={showErrorPanel}
-        confirmDialog={confirmDialog}
-        submitButtonText={submitButtonText}
-        confirmCloseIfNotSaved
-      />
-    ) }) }
-  </Modal>
+) => {
```

Comment:
It looks like nothing changed here functionally, so can you change this back to how it was?